frontend/themes/ce-demo/components/vaadin-dialog-overlay.css

```diff
@@ -1,7 +1,19 @@
 :host([theme~="no-padding"]) [part="content"] {
   padding: 0;
 }
-/* TODO: TEMPORARY FIX UNTIL https://github.com/vaadin/collaboration-engine-internal/pull/548 IS MERGED AND RELEASED */
-[part="overlay"] {
+
+:host([theme~="editor-view-dialog"]) [part="overlay"] {
+  height: 100%;
+  max-width: 400px;
+  width: 100%;
+  /* Temporary fix https://github.com/vaadin/collaboration-engine-internal/pull/548 */
   z-index: -1;
 }
+
+@media (min-width: 1024px) {
+  :host([theme~="editor-view-dialog"]) [part="overlay"] {
+    height: auto;
+    max-width: none;
+    width: 800px;
```

Comment:
On desktop, the height is now bigger than the width. I'm not sure what to think about that. It would feel more natural to have a wide dialog on a wide screen, and this is also how it's illustrated in Figma. But this is not a blocker for me. If you think this is better, I'm okay with that.

Reply:
Fixed width because:
- Input fields that are unnecessarily wide can be confusing.
- Wide paragraphs of text (comments) can be difficult to read.

No fixed height because it might cause unnecessary scrolling. We could perhaps set a max-height that still displays the entire form.

Apologies for the robotic nature of this comment. :D

Reply:
Good arguments, no objections. :)